src/Containers/VendorForm/VendorForm.js

```diff
@@ -1,34 +1,78 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { useDropzone } from 'react-dropzone'
+import request from 'superagent';
 import './VendorForm.scss';
 import { addVendors } from '../../actions';
 import images from '../../images/images';
 import VendorProductContainer from '../VendorProductContainer/VendorProductContainer';
 import { createVendor } from '../../apiCalls';
 
-
 export const VendorForm = () => {
   const [vendorName, setVendorName] = useState('');
   const [vendorDescription, setVendorDescription] = useState('');
   const [vendorImage, setVendorImage] = useState('');
   const [vendorProducts, setVendorProducts] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const CLOUDINARY_UPLOAD_PRESET = 'Farmer_Images';
+  const CLOUDINARY_UPLOAD_URL = 'https://api.cloudinary.com/v1_1/dlgdlli2u/image/upload';
 
-const handleCreateVendor = () => {
-  createVendor(vendorName, vendorDescription, vendorImage)
-    .then(response => {
-    console.log(response)
-    })
-  .catch(error => console.log(error))
-}
+  const handleImageUpload = (file) => {
+    let upload = request.post(CLOUDINARY_UPLOAD_URL)
```

Comment:
This looks like a great tool for the image upload for our app.  Thanks for implementing this code